### PR TITLE
Cursor tweaks

### PR DESCRIPTION
**Pull request by andrepolischuk**
@iamstarkov good?

### CODE REVIEW

**Comment by andrepolischuk** [comment]
I make some refactoring and insert colors used one time directly

File: index.js [hunk 1]
```diff
@@ -85,7 +83,10 @@ exports.decorateConfig = config => Object.assign({}, config, {
   termCSS: `
     ${config.termCSS || ''}
     .cursor-node {
-      mix-blend-mode: difference;
+      mix-blend-mode: multiply;
```
Comment (iamstarkov) [inline index.js]: whats this? ive never heard of this props

Comment (andrepolischuk) [inline index.js]: Cursor is multiplied by background and replaces the background color. Resulting color is always as dark as background.

Comment (iamstarkov) [inline index.js]: should be good

**Comment by iamstarkov** [comment]
👍

**Comment by andrepolischuk** [comment]
Published as 0.5.0

**Comment by iamstarkov** [comment]
got it already, thank you
screen shot 2016-08-17 at 15 29 09

**Pull request merged**